rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -682,7 +691,7 @@ void store_security_filepath_in_qos(
 }
 
 /*  Set all the qos properties needed to enable DDS security  */
-void configure_qos_for_security(
+rmw_ret_t configure_qos_for_security(
```

Comment:
Consider putting the `#if` inside the function, otherwise you have the opportunity to strangely diverge function signatures. In fact... you already have (check your return types).

Reply:
Updated.